lighthouse-core/report/v2/renderer/dom.js

```diff
@@ -37,7 +37,9 @@ class DOM {
       element.className = className;
     }
     Object.keys(attrs).forEach(key => {
-      element.setAttribute(key, attrs[key]);
+      if (attrs[key] !== undefined) {
```

Comment:
if it shows up in `Object.keys()` but is `undefined` then it would be `{!Object<string, (string|undefined)>=}` but it's still silly to be passing in undefined attributes. If it's just in that one spot, why not make a `const attributes = item.snippet ? {title: item.snippet} : undefined` or whatever there